### PR TITLE
Refactor images.py

### PR DESCRIPTION
**Pull request by bijij**
Untested so sue me, but it should make bytesIO work as expected?

### CODE REVIEW

File: cogs/images.py [hunk 1]
```diff
@@ -66,97 +66,89 @@ def _shifter(self, attachment_bytes: bytes, size: tuple):
...
-    def loop_jpeg(self, severity, filename, loopyloops):
-        for _ in range(loopyloops):
-            image = Image.open(filename)
-            image.save(filename, format='jpeg', quality=severity)
-            image.close()
+    def _get_image(self, ctx: NewCtx) -> Tuple[BytesIO, str, Tuple[int, int]]:
+        attachment_file = BytesIO()
 
+        if not ctx.message.attachments:
+            await ctx.author.avatar_url_as(size=1024, format='png').save(attachment_file)
```
Comment (sudosnok) [inline cogs/images.py:91]: what are the ytes being returned to?

File: cogs/images.py [hunk 2]
```diff
@@ -66,97 +66,89 @@ def _shifter(self, attachment_bytes: bytes, size: tuple):
...
+            filename = ctx.author.display_name + '.png'
+            file_size = (1024, 1024)
 
+        else:
+            target = ctx.message.attachments[0]
+            await target.save(attachment_file)
```
Comment (sudosnok) [inline cogs/images.py:97]: same here ,bytes going nowhere

File: cogs/images.py (hunk 1 above)
Comment (bijij) [reply cogs/images.py:91]: they're written to `attachment_file` the BytesIO object

File: cogs/images.py [hunk 3]
```diff
@@ -66,97 +66,89 @@ def _shifter(self, attachment_bytes: bytes, size: tuple):
 
         new_image = Image.merge('RGB', (new_red, new_green, new_blue))
 
-        output = BytesIO()
-        new_image.save(output, format="png")
-        output.seek(0)
-        
-        return output
+        out_file = BytesIO()
+        new_image.save(out_file)
```
Comment (sudosnok) [inline cogs/images.py]: Image.save from a file-like object requires the format to be specified

File: cogs/images.py (hunk 1 above)
Comment (sudosnok) [reply cogs/images.py:91]: woh,sorry, you right

**Pull request merged**